### DISCUSSION TITLE
Limit spanned traffic to DNS only

### DISCUSSION THREAD

**Discussion by jinksy31337** [Security-Onion-Solutions/securityonion#4142] [Unsupported Versions]
Does anyone know if its possible to limit what kind of packets SO logs, at the moment i have it spanned and logging all captured packets, however, i only want to log DNS traffic, does anyone know if it's possible?

**Comment by Shak3** [comment]
https://docs.securityonion.net/en/2.3/bpf.html
Follow this documentation and put your BPF as
steno:
bpf:
  - port 53

nids:
 bpf:
  - port 53

zeek:
 bpf:
  - port 53

in the file /opt/so/saltstack/local/pillar/global.sls

**Reply by Shak3** [reply]
not sure why its putting the dots... but they are dashes

**Comment by gcsrellis** [comment]
sudo so-zeek-logs will allow you to choose which zeek logs you want.  Just uncheck everything except DNS and you should be set!
